content/02-datamodeling/05-correctness-constraints.mdx

```diff
@@ -105,3 +105,41 @@ This has caused no end of confusion and will likely continue to do so in perpetu
 There are many more ways in which data may be incorrect than addressed here. Constraints help, but even they are only so flexible; many common intra-table specifications, like a limit of two or higher on the number of times a value is allowed to appear in a column, can only be enforced with [triggers](functional-units#triggers-and-consequences).
 
 But there are also ways in which the very structure of a table can lead to inconsistencies. To prevent these, we'll need to marshal both primary and foreign keys not just to define and validate but to *normalize* the relationships between tables. First, though, we've barely scratched the surface of how [the relationships between tables define the structure of the database itself](making-connections).
+
+## FAQ
+
+<details><summary>What is the definition of a tuple?</summary>
+
+A tuple is a data structure that stores a specific number of elements. These elements may include integers, characters, strings, or other data types.
+
+Tuples are static and cannot be modified, typically resulting in lower memory requirements than arrays.
```

Comment:
I'm curious if this definition of a tuple works in the context of databases.  For a tuple in programming languages like Python, these statements are true.  For a database, I think it's a bit looser if I'm not mistaken.  A tuple in databases is typically just a single row in a database, so the results can be modified I think.

Reply:
The third FAQ addresses tuple in a database directly where you are correct, it is more general.